lib/phlex/testing/nokogiri.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "nokogiri"
+
+module Phlex::Testing::Nokogiri
+	include Phlex::Testing::SGML
+
+	def render_fragment(...)
+		::Nokogiri::HTML5.fragment(
+			render_to_string(...),
+		)
+	end
+
+	def render_document(...)
+		::Nokogiri::HTML5(
+			render_to_string(...),
+		)
+	end
```

Comment:
In Nokogiri and Nokolexbor, you need to specify if you’re rendering a document or a fragment.